### ISSUE TITLE
'High Contrast Black' theme Menu WCAG AAA contrast compliance

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1010]
**Describe the bug**
The Menu components' various states color contrast ratio is not 7:1 or above.

> What version of the Fundamental Library Styles are you using?
v0.9.1-rc.0

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/menu.html
2. Check the Menu list with different states' color contrast

**Expected behavior**
If we want WCAG AAA compliance, this contrast ratio should be 7:1 or above.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81753823-801bbd80-9469-11ea-9720-74ee64e159aa.png)
![image](https://user-images.githubusercontent.com/43764832/81753968-da1c8300-9469-11ea-8acc-3bd4f2de9a25.png)
![image](https://user-images.githubusercontent.com/43764832/81754027-f9b3ab80-9469-11ea-8061-6e4bcd524b13.png)
![image](https://user-images.githubusercontent.com/43764832/81754068-14862000-946a-11ea-8a50-7362d60960fc.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)
